ixdiagnose/plugins/vm.py

```diff
@@ -1,10 +1,23 @@
-from ixdiagnose.utils.middleware import MiddlewareCommand
+import typing
+
+from ixdiagnose.utils.formatter import remove_keys
+from ixdiagnose.utils.middleware import MiddlewareClient, MiddlewareCommand
 
 from .base import Plugin
-from .metrics import FileMetric, MiddlewareClientMetric
+from .metrics import FileMetric, MiddlewareClientMetric, PythonMetric
 from .prerequisites import VMPrerequisite
 
 
+def vm_query(client: MiddlewareClient, context: typing.Any) -> list:
```

Comment:
Why can't you just do `remove_key(['devices.attributes.password'])`?

Reply:
We can simply this for loop and make it perform better in the process. I'm thinking of doing something like
```
for i in query_output:
    try:
        i['devices']['attributes']['password'] = None
    except KeyError:
        continue

return query_output